### DISCUSSION TITLE
File path of the .exe program?

### DISCUSSION THREAD

**Discussion by ZoroasterConor** [bitcraze/discussions#374] [Q&A] [answered]
Hi! I used pip3 to install cfclient on my windows pc but I can't find the .ese program to open the application. I used the pip3 list -v command but in the location it returns for cfclient I saw no program to open. Where should I be looking at? Thank you.

**Comment by knmcguire** [comment, marked as answer]
Hi!

It really depends if you used an python environment or not (we really recommend doing so). If not, than the executable of the cfclient should be found on something like:
_C:\Users\USERNAME\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\Scripts_

**Reply by ZoroasterConor** [reply]
Thank you! It is indeed there.
